test/es6/proxyenumremoval.js

```diff
@@ -90,7 +90,7 @@ var already_non_enmerable = false;
 var getPrototypeOfCalled = 0;
 var proxy = new Proxy({}, {
   ownKeys: function() {
-    return ['a','b','a']; // make first a non-enumerable, and second a enumerable, second a won't show up in for-in
+    return ['a','b']; // make first a non-enumerable, and second a enumerable, second a won't show up in for-in

```

Comment:
Seems like this comment is no longer relevant, and the code related to already_non_enumerable could also be removed